packages/fxa-settings/src/components/ConnectedServices/index.test.tsx

```diff
@@ -254,9 +268,14 @@ describe('Connected Services', () => {
     ).length;
     await clickFirstSignOutButton();
     await clickConfirmDisconnectButton();
+    expect(logViewEvent).toHaveBeenCalledWith(
+      'settings.clients.disconnect',
+      'submit.'
+    );
     const finalCount = (
       await screen.findAllByTestId('settings-connected-service')
     ).length;
+    // TODO: fix this test, it no workey
```

Comment:
I think this is due to `clickFirstSignOutButton()`. Per our discussion, this does not affect your PR functionality. We can update this at a later time.

Reply:
Thanks for looking into it and calling it out, this PR is a couple months old so I'd forgotten about this. I created a follow up for it and will reference it on this TODO line: FXA-4453 / #11658